food-for-thought/app/changepassword.tsx

```diff
@@ -88,6 +88,10 @@ const ChangePassword = () => {
           <Icon name="lock" size={20} color="#7E7093" style={styles.icon} />
         </View>
 
+        {errorMessage ? (
+          <Text style={styles.errorMessage}>{errorMessage}</Text>
+        ) : null}
```

Comment:
This code is all good, just FYI can do `{errorMessage && <Text...}` just a little neater! this only renders `<Text>` if `errorMessage` exists. But will need to change default to `setErrorMessage(undefined)` instead of `setErrorMessage('')` - usually using `undefined` is better practice than `''`

Reply:
Awesome tysm, will fix now!